.travis.yml

```diff
@@ -8,8 +8,11 @@ go_import_path: github.com/moolitayer/awx-client-go
 
 install:
 - go get github.com/golang/glog
+- go get golang.org/x/tools/cmd/goimports
```

Comment:
Can you please add to the README, "Installation section"
go get golang.org/x/tools/cmd/goimports # Test dependency

Reply:
Done.